### PR TITLE
Use firebase images

### PR DESCRIPTION
**Pull request by ganhongyao**

### CODE REVIEW

File: backend/app/models/primary_donor.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # frozen_string_literal: true
 
 class PrimaryDonor < ApplicationRecord
-  has_one_base64_attached :image
-
   has_many :campaigns, dependent: :destroy
 
   validates :name, presence: true
   validates :email, presence: true, uniqueness: true, format: { with: URI::MailTo::EMAIL_REGEXP }
+  # validates :image_url, presence: true, allow_blank: false
```
Comment (ganhongyao) [inline backend/app/models/primary_donor.rb:8]: These validations + database constraints should be added back once we have migrated the data over.

**Review by sivayogasubramanian — COMMENTED** [review]
- [ ] Remove Base64String type file
- [ ] Remove `helper Base64Helper` in application_controller.rb
- [ ] Delete the helper file
- [ ] Remove all active_storage dependencies
- [ ] Update deployment to remove file_storage volume (line 24 and 25 of docker-compose.yml)

File: frontend/pages/admin/charities/index.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ const AdminCharities = () => {
     setSelectedCharity(charity);
   };
 
-  const createLogos = (logoBase64: Base64String) => logoBase64 && <Box component="img" src={logoBase64} sx={logoSx} />;
+  const createLogos = (logoUrl: Base64String) => logoUrl && <Box component="img" src={logoUrl} sx={logoSx} />;
```
[the remark this answers is not in the document]
Comment (ganhongyao) [reply frontend/pages/admin/charities/index.tsx:35]: Keeping `Base64String` as it is still used in `compressImageThenConvertToBase64String`.

[the remark this answers is not in the document]
Comment (ganhongyao) [reply frontend/pages/admin/charities/index.tsx:35]: @sivayogasubramanian Removed the active storage tables as well.

File: backend/app/models/primary_donor.rb (hunk 1 above)
Comment (sivayogasubramanian) [reply backend/app/models/primary_donor.rb:8]: How will we be migrating the data?

**Review by sivayogasubramanian — APPROVED** [review]
LGTM!

**Pull request merged**